server/controllers/deleteDraftLocation/confirm.ts

```diff
@@ -0,0 +1,63 @@
+import { NextFunction, Response } from 'express'
+import FormWizard from 'hmpo-form-wizard'
+import backUrl from '../../utils/backUrl'
+import { TypedLocals } from '../../@types/express'
+import FormInitialStep from '../base/formInitialStep'
+
+export default class ConfirmDeleteDraftLocation extends FormInitialStep {
+  override locals(req: FormWizard.Request, res: Response): Partial<TypedLocals> {
+    const locals = super.locals(req, res)
+    const { decoratedResidentialSummary } = res.locals
+    const { id: locationId, prisonId } = decoratedResidentialSummary.location
+    const locationType = decoratedResidentialSummary.location.locationType.toLowerCase()
+
+    const backLink = backUrl(req, {
+      fallbackUrl: `/view-and-update-locations/${prisonId}/${locationId}`,
+    })
+
+    return {
+      ...locals,
+      backLink,
+      cancelLink: `/view-and-update-locations/${prisonId}/${locationId}`,
```

Comment:
I don't think we need to explicitly set backLink and cancelLink anymore, can you try removing them and see if both links still go to the same place?